File: src/main/java/com/elbertribeiro/BookApi.java
package com.elbertribeiro;

import org.springframework.boot.*;
import org.springframework.boot.autoconfigure.*;
import org.springframework.web.bind.annotation.*;

@SpringBootApplication
@RestController
public class BookApi {

    public static void main(String[] args) {
        SpringApplication.run(BookApi.class, args);
    }

    @GetMapping("/")
    String home() {
        return "Spring is here!";
    }
}

File: src/main/kotlin/com/elbertribeiro/BookApi.kt
package com.elbertribeiro

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RestController

@SpringBootApplication
@RestController
class BookApi {
    @GetMapping("/")
    fun home(): String {
        return "Spring is here!"
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(BookApi::class.java, *args)
        }
    }
}